docs/cugraph/source/conf.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2018-2023, NVIDIA CORPORATION.
+# Copyright (c) 2018-2024, NVIDIA CORPORATION.
```

Comment:
These copyright changes on files I didn't touch were all pulled in by:

```shell
pre-commit run --all-files
```

This file was last touched January 18, 2024 (https://github.com/rapidsai/cugraph/commits/branch-24.04/docs/cugraph/source/conf.py), so I guess this makes sense:

<img width="1318" alt="image" src="https://github.com/rapidsai/cugraph/assets/7608904/542daaff-4425-4730-a020-40ac40e84c78">

But I don't really understand why that wouldn't have been caught in the other PRs targeting this branch since then, like #4109 or #4148.

Doesn't really bother me, but just noting (cc @KyleFromNVIDIA) in case that indicates that there's something missing in how the copyright-update check + hook work.

Reply:
The new copyright script isn't being used yet. https://github.com/rapidsai/cugraph/pull/4130 adds it but hasn't been merged yet.

Reply:
oooo should have noticed that PR, sorry 🙈